plugins/@grouparoo/google-sheets/__tests__/.env.example

```diff
@@ -0,0 +1,3 @@
+# Note: These are only needed when recording new nock fixtures
+GOOGLE_SERVICE_CLIENT_EMAIL = "grouparoo-sample@sample-sources.iam.gserviceaccount.com"
+GOOGLE_SERVICE_PRIVATE_KEY = "-----BEGIN PRIVATE KEY-----\nMIIEvQIBADANBgkqhkiG9w0BAQEFAASCBKcwggSjAgEAAoIBAQC8gJERtBcVnKat\nssKk+5XnIy5m7xZmmVdWTWl2Xdzb4ilVPnxyTzN4iahibg0QfJ5xFMpp4zDuoLrJ\nuNXp109XK0rpKl4HhHE/cG3cfSbQWvREl1x3qSyfcM/XUEzk7hfhjOAkVc7i/nyY\nRWmrS+ynhcH40OiqTEt4S/ceMXfKaGJHDeezN8dfAdNimvo/5W6uADPwuMEy8gsF\n5/Itl7XwtASLftQ6QRS0R6IQffiW8PObYyuJ7Ll1l7hvuUW6EaKUc2CQZ4Uw9sHB\n050+AVB0cKtDQw8H2aruDs7rd6YwGP1JjBP1NG/Tz5hDF7QMtOC+QzIQt2afOx9w\nCggkNVFTAgMBAAECggEAROdqH+KInanJc6zoYY5lWTwO9ilnMtHSjwV1C4N7/BRL\nr5FLxC0XT3RiTLiVpfUnpYZIBzS46kFuhdY+u1+s7NwyYdDO1kifbOUYA4Tr7zWQ\nHJKHhmYcnZuLHdWIhhZSPGxWcnG7UlZcDP0yhx8VvpayNNWIdte0rqB2EF9cvmuQ\nJidlbi2FLKifTmKhdtqIoXSHbg5Bp7dcVJAhMwf6u1/AJixqclTMYDs7Yp7XvGfQ\nC2t+YYAytEJaDaeNqm3DGWh/Mmtzwau7x1C1lMsrLyOlSxjKHsIoRQZU1bxzAx0M\n2+FL9mclHof6Ml0+FtuYYDHxTPDaKyEK+dv+71OAQQKBgQDz7FjhsgMJE18UcXkN\nbi03lH1tUZqyVgZlrblK0v4uDMZHN71xDAomleqcJAhT9L2KIv+VulhkCojxZ3eu\nyvov/LoRKIZvHIFnChT/YzLiiGiV2CWtRydmCFxwaQEdemt2j9Nk0EpschBnGKbj\n34WEPYgFWCb06Mvbqb+v90Pk+wKBgQDF1cZh48QTCRrx/XcTJUlBMAZrzbEEwQo1\nRI6u199xXuEteDX9PzwznGwzpBxzp+QHl6P7k36+x6HWolW60Y+WbAABBeEbEa94\n3Phl+IyES7UcZjg2dPL+2M1+DgMHaR7t2/ra6xbxfxnYs4etS6TrGj8c1OyGyUA5\nA7nblL6liQKBgBfcd/lkG43oujVawFfC/mNggECQagymA6wwqV6bc0f38fGHj12e\nquOJaNDraXElEOGReOUhvIHU0f6je3661myZUW5aQ4RdCn+U6ebtwk2MwIZgBZeE\nzWLa0IgijXxfvGF1wl0rTwx6wAYF0FWcSQhdqI+1RuJQX5jWay0AiEC/AoGAGMSr\n4qOOb8erDPLrzoTTiArIMwT8jZjRe1vK9U7nlZ0IWhX7xGn/UoLZ9RktyrkRTnYI\nvL15l88Mjv+h9hK4hF+fHIGs7iOnsV0Q6EtW5FVKJnURwpefuJJSH98HOA1MJwFs\nha37IF8GUGAhs2uOCTUvA6AQwMs/dCPoR/p2Z+kCgYEAh0f8w/SRQKspwmi7aW7X\nWXSxCUcwQFT80prsja0LdUb7uhaBcDx+cL54fTds6QGo/P4FMuAw/1JBqo8r9vtb\nSF1B6mKBb68CoCafriHh1i64/IRCR585G0Mzzva5Z+hkn0250YfJxazLVtG1rMwK\nwoJ3raNOHAwD7SYipuX+6t8=\n-----END PRIVATE KEY-----"
```

Comment:
Has this account/key has been deactivated?

Reply:
i made that up randomly.